packages/api/tests/lambdas/test-publish-reports.js

```diff
@@ -744,13 +732,20 @@ test.serial('publish failure to PDRs topic does not affect publishing to other t
 test.serial('handler publishes notification from input to first failed Lambda step in failed execution history', async (t) => {
   const granulePublishMock = publishReports.__set__('publishGranuleSnsMessage', granulePublishSpy);
 
-  const cwEventMessage = createCloudwatchEventMessage(
-    'FAILED',
-    createCumulusMessage({
-      numberOfGranules: 1
+  const message = createCumulusMessage({ numberOfGranules: 1 });
+
+  await executionModel.create(
```

Comment:
I had to double check and remind myself what this test is even doing to be sure that *not finding an existing execution* wasn't part of what we're intending to test, but it's not. Same for the rest of the changes. So thanks for catching these 👍